pinc/js_newwin.inc

```diff
@@ -8,32 +8,20 @@ include_once($relPath.'resolution.inc');
 
 function prep_for_links_to_project_pages()
 {
-    global $userP, $i_resolutions, $code_url;
-
-    // Ensure that the guts of this function are only done once,
```

Comment:
We still need this logic in here to prevent the function from being called twice and two instances of this javascript function being included in the page.

Alternatively we could make this function return the javascript and put that in the js_data part of the `output_header()` call, but that would be a more intrusive change and I'm fine if we just put the don't-duplicate-output back in.

Reply:
Done this.